### DISCUSSION TITLE
Connection names in Speedify UI not corresponding to USB names

### DISCUSSION THREAD

**Discussion by CoreyJaouen** [SmoothWAN/SmoothWAN#61] [General & Help]
I have renamed the connections in the USB setup to their corresponding network names but the Speedify UI ignores them and instead either displays them both as "Starlink" or both as "Verizon Wireless". 

It previously worked but something went wrong and the PI would not boot completely (red light stuck on, maybe bad power adapter?) and this is the latest hiccup since reflashing the sd card. I've tried reflashing the sdcard with the latest build multiple times but still getting the same issue. 

Any ideas? Thank you so much for smoothwan!

**Comment by TalalMash** [comment]
Hello,

I've contacted Speedify about having the option to use interface names instead of ISP names, they mentioned that will probably add the interface name somewhere more visible in the next versions but no options to revert to interface name only like the older versions.

Regarding the boot issue, have you tried using a different SD card to isolate the issue?

**Reply by CoreyJaouen** [reply]
Thanks for the quick reply. I'm guessing that's related to the new "Automatic Priority" feature? It's just odd that it displays both connections as the same ISP rather than properly identifying each one separately. Makes it kinda confusing when monitoring. 

As for the SD card I've ordered a "SanDisk Ultra SDSQUNS-032G-GN3MN 32GB" so I'm waiting on that. Currently using a 16gb Kingston.

**Reply by TalalMash** [reply]
Yes I'm having the same issue only with the latest version as well as two reports on the Discord channel so far, I've added a Generate Logs button in the Speedify section if the support team ask for it.
